server/game/Clocks/Clock.js

```diff
@@ -8,6 +8,20 @@ class Clock {
         this.paused = false;
         this.stateId = 0;
         this.name = 'Clock';
+        this.delayToStartClock = delayToStartClock;
+        this.manuallyPaused = false;
+    }
+
+    manuallyPause() {
+        this.stop();
+        this.manuallyPaused = true;
+        this.updateStateId();
+    }
+
+    manuallyResume() {
+        this.timerStart = 0;
+        this.manuallyPaused = false;
+        this.updateStateId();
```

Comment:
As above, manually resuming the clocks now resets all timing variables so that the game doesn't realize that any time has elapsed since they were paused.